client/my-sites/comments/comment-list/index.jsx

```diff
@@ -162,6 +163,7 @@ export class CommentList extends Component {
 
 		return (
 			<div className="comment-list">
+				<QuerySiteComments siteId={ siteId } status="all" />
```

Comment:
I also moved this from main into the block component (in case we drop this into any other section). Eventually I think we'd want to update the status param to reflect the current status but that can wait for later.